Makefile

```diff
@@ -91,9 +91,12 @@ verify-codegen:
 .PHONY: verify-vendor-licenses
 verify-vendor-licenses:
 	hack/verify-vendor-licenses.sh
-.PHONY: verify-crds
-verify-crds:
-	hack/verify-crds.sh
+# .PHONY: verify-crds
```

Comment:
ditto

Reply:
apologies, verify-crds is not working properly, I had to remove it to run make-verify which was after it. I forgot to put it back